src/Voron/Platform/Posix/AllowBuildOnWindows.cs

```diff
@@ -494,7 +494,9 @@ public enum OpenFlags : int
         O_TRUNC = 0x00000200,
         O_APPEND = 0x00000400,
         O_NONBLOCK = 0x00000800,
-        O_SYNC = 0x00001000,
+        O_SYNC = 1052672, // 0x00101000, // value directly from printf("%d", O_SYNC)
+        O_DSYNC = 4096, // 0x00001000, // value directly from printf("%d", O_DSYNC)
+        O_DIRECT = 16384, // 0x00004000, // value directly from printf("%d", O_DSYNC)
```

Comment:
I didn't commit the change by mistake. But I used it in the test.  Will be pushed in few minutes